### DISCUSSION TITLE
Are there extra settings/options for the biometric unlock feature?

### DISCUSSION THREAD

**Discussion by WGandy** [astubenbord/paperless-mobile#278] [Q&A]
I just turned this on and there don't appear to be any other settings than that. But I'm wondering if features like locking the app after a period of time that the app isn't in the foreground might be possible. Currently, it seems that flicking the app away (closing it) is the only way to lock it short of powering the device off. I would like to know that it's locked when my screen locks AND/OR after 15 minutes of inactivity.  I would also like to see a different authentication method being added (something separate from the device screen lock if it's using bio). 

I'm truly surprised that there aren't more security features built into stock android AND that people don't seem to be concerned about their documents being so easily available if someone takes their mobile device and simply figures our their pin or face id. And I think there are other ways of cracking the screen locking in android using specialty software too. It's one thing if they can get your angry bird gaming history but think about if they can access all of your scanned documents! I'd like to see more security features for the app. 

All that said, I'm very impressed with the quality of the app. It's very smooth, fast and pretty intuitive. I look forward to using it a lot more in the future.

**Comment by florianschroen** [comment]
Most normal apps get stopped after display off by the power saving mechanism of the mobile phones on Android.

For iOS I don't know.

If you do not trust the system provided (biometric) lock mechanisms, you should care about all your other apps/services/data, too. Especially if there are cloud accounts. So you have to think about disabling the unsecured unlocking types. 

Apps can rely on the biometric options provided by the system. This is a yes or no option. So at this point no additional setting will be needed.

But I agree with you, that the app would need an internal counter for self locking after inactivity (being in background). Other apps do this too. eg keepass2android.

For this a config option for the self lock timeout will be preferred to a hardcoded value.

**Reply by astubenbord** [reply]
That's something I could do, but I'm pretty overloaded at the moment given the speed at which the paperless-ngx team makes progress and implements new features. This is a rather low priority issue, at least for me.

**Reply by florianschroen** [reply]
I agree with low prio.

I appreciate you are will to implement it.
